ambry-quota/src/main/java/com/github/ambry/quota/QuotaUtils.java

```diff
@@ -83,7 +83,7 @@ public static QuotaChargeCallback buildQuotaChargeCallback(RestRequest restReque
     if (!quotaManager.getQuotaConfig().bandwidthThrottlingFeatureEnabled) {
       return new PostProcessQuotaChargeCallback(quotaManager, restRequest, isQuotaEnforcedOnRequest);
     } else {
-      throw new UnsupportedOperationException("Not implemented yet.");
+      return new PreProcessQuotaChargeCallback(quotaManager, restRequest);
```

Comment:
I suppose this is the real implementation for bandwidth throttling. We will move to this implementation later, right?